### PR TITLE
Add link to prom2teams

### PR DESCRIPTION
**Pull request by dortegau**
Adding prom2teams link :)

@brian-brazil

### CODE REVIEW

File: content/docs/operating/integrations.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Phabricator / Maniphest](https://github.com/knyar/phalerts)
   * [SMS](https://github.com/messagebird/sachet): supports [multiple providers](https://github.com/messagebird/sachet/blob/master/examples/config.yaml)
   * [Telegram bot](https://github.com/inCaller/prometheus_bot)
+  * [prom2teams](https://github.com/idealista/prom2teams)
```
Comment (brian-brazil) [inline content/docs/operating/integrations.md]: It's not obvious to me what this talks to, "teams" is a very generic term. Can you clarify?

Comment (dortegau) [reply content/docs/operating/integrations.md]: Added short description to clarify :)

Comment (brian-brazil) [reply content/docs/operating/integrations.md]: That's a bit too wordy now, we just need to get the phrase "Microsoft Teams" in there

Comment (dortegau) [reply content/docs/operating/integrations.md]: I've removed some words to make it less verbose

File: content/docs/operating/integrations.md [hunk 2]
```diff
@@ -50,7 +50,7 @@ For notification mechanisms not natively supported by the Alertmanager, the
   * [Phabricator / Maniphest](https://github.com/knyar/phalerts)
   * [SMS](https://github.com/messagebird/sachet): supports [multiple providers](https://github.com/messagebird/sachet/blob/master/examples/config.yaml)
   * [Telegram bot](https://github.com/inCaller/prometheus_bot)
-  * [prom2teams](https://github.com/idealista/prom2teams): forwards received alerts to [Microsoft Teams](http://teams.microsoft.com) using defined connectors
+  * [prom2teams](https://github.com/idealista/prom2teams): forwards alerts to Microsoft Teams
```
Comment (brian-brazil) [inline content/docs/operating/integrations.md]: notifications, not alerts.

Keep it sorted.

Comment (dortegau) [reply content/docs/operating/integrations.md]: done

**Comment by brian-brazil** [comment]
Thanks!
